bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala

```diff
@@ -94,7 +94,7 @@ class PartnerView extends BorderPanel with BoundView[Proposal] {view =>
       BorderFactory.createEmptyBorder(2, 4, 2, 4))
     override def refresh(m:Option[Proposal]) {
 
-      val (reqTime, minTime) = ~m.map(_.proposalClass).map(r => (r.requestedTime.hours, r.minRequestedTime.hours))
+      val (reqTime, minTime) = m.map(_.proposalClass).map(r => (r.requestedTime.hours, r.minRequestedTime.hours)) | ((0.0, 0.0))
```

Comment:
I think it is there if you can figure out how to use the right imports.  It seems that 

````scala
import scalaz._, Scalaz._
````

is no longer always sufficient or correct.  I just punted.